backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.json

```diff
@@ -25,5 +25,8 @@
     "Issuer": "amui",
     "App": "access-management-ui",
     "CertificateName": "JWTCertificate"
+  },
+  "LogicFlags": {
+    "DisplayPopularSingleRightsServices": false
```

Comment:
Despite what we talked about earlier, this ended up kinds feature-flaggy anyway... 😅
I tried giving it a name that didn't scream feature flag though, if that makes sense, but maybe that's a mistake too? What do you think?

Reply:
Yeah, this seems like a good solution for now!
Since it's essentially controlling the visibility of a feature, it might be clearer to just embrace the term "feature flag" and rename it accordingly.

I still think we probably need the ability to quickly enable/disable the functionality without requiring code deployments in the future. but we can implement that at a later time. 
👍

Reply:
I agree on both counts - I'll rename this and then we can add a more sophisticated solution later 😊